framework_tvb/tvb/interfaces/rest/server/resources/util.py

```diff
@@ -29,16 +29,33 @@
 #
 
 import os
+import shutil
 import tempfile
 from tvb.basic.profile import TvbProfile
 from werkzeug.utils import secure_filename
 
 
-def save_temporary_file(file):
+def save_temporary_file(file, destination_folder):
```

Comment:
This `utils.py` module is not very nice and OOP-friendly. Can we consider putting these functions in another place? e.g. in the base `RestResource` class?